### PR TITLE
Use new matrix builds Circle CI feature

### PR DESCRIPTION
**Pull request by sjmiller609**
also, add kube 1.18 to tests

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -94,113 +54,29 @@ jobs:
...
+    parameters:
+      executor:
+        type: string
+        default: "LocalExecutor"
+      kube_version:
+        type: string
+        default: "1.18.2"
     environment:
       KUBE_VERSION: 1.14.10
       EXECUTOR: "LocalExecutor"
```
Comment (kaxil) [inline .circleci/config.yml:69]: We can remove this now I think

Comment (sjmiller609) [reply .circleci/config.yml:69]: oops, good catch

Comment (kaxil) [reply .circleci/config.yml]: or use parameters here:

```
```suggestion
      KUBE_VERSION: << parameters.kube_version >>
      EXECUTOR: << parameters.executor >>
```

Comment (sjmiller609) [reply .circleci/config.yml]: idk if that would work or not though

**Pull request merged**